safe_transaction_service/events/services/queue_service.py

```diff
@@ -137,15 +132,15 @@ def send_unsent_events(self) -> int:
         self.unsent_events = []
 
         total_sent_events = 0
-        logger.info("Sending %i not sent messages", len(unsent_events))
+        logger.info("Sending previously unsent messages: %i", len(unsent_events))
         for unsent_message in unsent_events:
             if broker_connection.publish(unsent_message):
                 total_sent_events += 1
             else:
                 self.unsent_events.append(unsent_message)
 
         self.release_connection(broker_connection)
-        logger.info("Sent %i not sent messages", total_sent_events)
+        logger.info("Correctly sent previously unsent messages: %i", total_sent_events)
```

Comment:
You are sending both the message you wanted to send and the unsent messages, message is confusing

```suggestion
        logger.info("Correctly sent messages: %i", total_sent_events)
```